recipes/gdcm/all/patches/0008-3.0.20-find-libuuid.patch

```diff
@@ -5,7 +5,7 @@
  if(GDCM_USE_SYSTEM_UUID)
    # If user say so, then this is a requirement !
 -  find_package(UUID REQUIRED)
-+  find_package(libuuid REQUIRED)
++  find_package(LibUUID REQUIRED)
```

Comment:
no need to change patches, libuuid is config file name of util-linux-libuuid, and libuuid::libuuid is a valid target name of this recipe